dbfit-java/core/src/main/java/dbfit/api/DbStoredProcedureCall.java

```diff
@@ -52,14 +51,8 @@ void bindParametersTo(StatementExecution cs) throws SQLException {
     }
 
     public StatementExecution toStatementExecution() throws SQLException {
-        String sql = toSqlString();
-        PreparedStatement ps = environment.getConnection().prepareCall(sql);
-        StatementExecution cs;
```

Comment:
May be we should rename `cs` now. Presumably `cs` was once a `CallableStatement`.